lib/run/index.js

```diff
@@ -26,6 +26,7 @@ var fs = require('fs'),
         request: ['response', 'request', 'item', 'cookies'],
         beforeTest: ['events', 'item'],
         test: ['executions', 'item'],
+        io: ['trace', 'response', 'request', 'cookies'],
```

Comment:
We might have to rethink `io` event. This variadic property is causing a pain for all consumers. App as well as newman.